frontend/src/core/hotkeys/hotkeys.ts

```diff
@@ -387,13 +387,31 @@ const DEFAULT_HOT_KEY = {
     name: "Create a cell before current cell",
     group: "Command",
     key: "a",
-    editable: false,
   },
   "command.createCellAfter": {
     name: "Create a cell after current cell",
     group: "Command",
     key: "b",
-    editable: false,
+  },
+  "command.copyCell": {
+    name: "Copy cell",
+    group: "Command",
+    key: "c",
+  },
+  "command.pasteCell": {
+    name: "Paste cell",
+    group: "Command",
+    key: "v",
+  },
+  "command.moveToTopCell": {
+    name: "Move to top cell",
+    group: "Command",
+    key: "Mod-ArrowUp",
```

Comment:
we already have `global.focusTop` and `global.focusBottom` that are configurable. I think we don't need to make keyboard nav arrow up/down configurable